src/services/requestHandler.js

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+
+/**
+ * This function creates an instance of axios and return
+ * it for making API request
+ * @returns { instance } an instance of axios
+ */
+const axiosInstance = () => {
+  let apiUrl = 'https://authorhaven.herokuapp.com/api';
```

Comment:
Instead of doing this. What do you think about using an environmental variable like process.env.BASE_URL